script/make_release.py

```diff
@@ -47,13 +49,53 @@ def get_latest_changelog_entries(path):
     return new_changelog_section
 
 
-def get_release_post(version, path):
-    release_post_header = f"---\ncategories:\n  - Release\ntags:\n  - gh-pages\n\ntitle: {version}\n---\n\n"
-    release_post_headline = "{{page.title}} is live and ready for [download](https://github.com/MaibornWolff/codecharta/releases/tag/{{page.title}}). This version brings the following:\n\n"
+def get_release_post(version, prefix_version, path, repository):
+    release_post_header = f"---\ncategories:\n  - Release\ntags:\n  - gh-pages\n  - release\n  - {repository.lower()}\n\ntitle: {repository} version {version}\n---\n\n"
+    release_post_headline = "{{page.title}} is live and ready for"+f" [download](https://github.com/MaibornWolff/codecharta/releases/tag/{prefix_version}). \nThis version brings the following:\n\n"
     release_post_content = get_latest_changelog_entries(path)
     return release_post_header + release_post_headline + release_post_content
 
 
+def update_changelog(changelog_path):
+  with in_place.InPlace(changelog_path, encoding="utf-8") as fp:
+    for line in fp:
+        if "## [unreleased]" in line:
+            fp.write(
+                f"## [{new_version}] - {date_formatted}\n")
+        else:
+            fp.write(line)
+
+  with in_place.InPlace(changelog_path, encoding="utf-8") as fp:
+    line_number = 0
+
+    for line in fp:
+        if line_number == 6:
```

Comment:
This seems to be ok, because it is unlikely that anybody adds something before line 6.

Reply:
I didn't change this. It is as before.